website/docs/commands/publishing/publish.md

```diff
@@ -81,6 +81,17 @@ To override this default value, set the `publish.computeVersion` directive, like
 //> using publish.computeVersion git:tag
 ```
 
+Please note that only tags that follow the semantic versioning are taken into consideration.
+
+Values available for project version configuration are:
+- `git:tag` or `git`: use the latest stable git tag, if it is older than HEAD then try to increment it
+  and add a suffix `-SNAPSHOT`, if no tag is available then use `0.1.0-SNAPSHOT`
+- `git:dynver`: use the latest (stable or unstable) git tag, if it is older than HEAD then use the output of
+  `-{distance from last tag}-g{shortened version of HEAD commit hash}-SNAPSHOT`, if no tag is available then use `0.1.0-SNAPSHOT`
+- `command:cmd`: use the output of the command `cmd` as version
```

Comment:
```suggestion
- `command:cmd`: use the output of the command `cmd` as version. It runs the cmd in your shell under the hood. This is not the recommended way since the downloaded script can contain any commands
```